Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -451,7 +453,7 @@ class PushNotificationIOS {
    * If you do not call this method your background remote notifications could
    * be throttled, to read more about it see the above documentation link.
    */
-  finish(fetchResult: FetchResult) {
+  finish(fetchResult: string) {
```

Comment:
Is this intentional. Why?

Reply:
Made a comment on the PR but probably should've mentioned it inline too.

> I also corrected a Flow type error - finish takes a string, not a FetchResult (which is the object containing the valid strings).

This type is incorrect, and it shows when you attempt to use the API.